clientv3/kv.go

```diff
@@ -43,6 +43,10 @@ type KV interface {
 	// if the required revision is compacted, the request will fail with ErrCompacted .
 	// When passed WithLimit(limit), the number of returned keys is bounded by limit.
 	// When passed WithSort(), the keys will be sorted.
+	// By default, Get returns results in lexiographical, ascending order of keys.
+	// When WithSort is specified, client fetches the entire key-space first, and then
+	// applies sorting or limits. For ascending ordered and limited results, one might
+	// omit WithSort option for better performance.
```

Comment:
Yeah how about if the limit is not specified and WithSort(ascending) is passed, just ignore the WithSort, so that we don't need this detail?